dev/AppNotifications/AppNotificationBuilder/AppNotificationComboBox.cpp

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "AppNotificationComboBox.h"
+#include "Microsoft.Windows.AppNotifications.Builder.AppNotificationComboBox.g.cpp"
+#include "AppNotificationBuilderUtility.h"
+
+namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
+{
+    AppNotificationComboBox::AppNotificationComboBox(hstring const& id) : m_id(id) { };
+
+    winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationComboBox AppNotificationComboBox::AddItem(winrt::hstring const& id, winrt::hstring const& content)
+    {
+        THROW_HR_IF_MSG(E_INVALIDARG, m_items.Size() >= c_maxSelectionElements, "Maximum number of items added");
```

Comment:
We should make sure that id and content are not empty here

Reply:
The is a check for ID not being empty on the line bellow.

Whether content should be left empty or not should be addressed when we get to this: https://microsoft.visualstudio.com/OS/_workitems/edit/40746502